dist/htmlminifier.js

```diff
@@ -1351,6 +1362,9 @@
         }
         if (options.minifyJS && isExecutableScript(currentTag, currentAttrs)) {
           text = minifyJS(text, options.minifyJS);
+          if (text.charAt(text.length - 1) === ';') {
+            text = text.slice(0, -1);
```

Comment:
That returns a whole match object, whereas `/;$/.test(text)` returns a boolean.

Anyway, I can't see how the RegExp version would give any performance advantage here.